wp-content/themes/pub/wporg-learn-2024/package.json

```diff
@@ -7,7 +7,9 @@
 	"private": true,
 	"devDependencies": {
 		"@wordpress/scripts": "27.2.0",
-		"rtlcss-webpack-plugin": "4.0.7"
+		"rtlcss-webpack-plugin": "4.0.7",
+		"@wordpress/icons": "^10.2.0",
+		"react-dom": "^18.3.1"
```

Comment:
I'm not 100% on this but I think there'll be a WP package for this, maybe `wordpress/element`?

Reply:
Tried in my IDE and looks good